cypress/fixtures/movie_data.json

```diff
@@ -0,0 +1,404 @@
+{
+  "movies": [
+    {
+      "id": 694919,
+      "poster_path":
+        "https://image.tmdb.org/t/p/original//6CoRTJTmijhBLJTUNoVSUNxZMEI.jpg",
+      "backdrop_path":
+        "https://image.tmdb.org/t/p/original//pq0JSpwyT2URytdFG0euztQPAyR.jpg",
```

Comment:
@zacwalters4 I see this is being used a `fixture` for the `movie details tests` to stub our network request. I haven't seen it be used before, so it was cool to see it in our tests. Thank you!